play-2.4/swagger-play2/build.sbt

```diff
@@ -6,6 +6,8 @@ checksums in update := Nil
 scalaVersion:= "2.11.6"
 crossScalaVersions := Seq("2.11.6", "2.11.7")
 
+resolvers += "scalaz-bintray" at "https://dl.bintray.com/scalaz/releases"
```

Comment:
@JosephEarl thanks! Can you try removing the resolver and updating specs2 to the latest `3.6.6` and see if that fixes it? See https://github.com/etorreborre/specs2/issues/295